Makefile

```
@@ -9,8 +9,78 @@ STYLE_CHECK_FILES=SimPEG examples tutorials tests
 # Since we are using the --ignore option we are overriding them. They are
 # included in the list so we keep ignoring them while running the
 # flake-permissive target.
-FLAKE8_IGNORE=E121,E123,E126,E226,E24,E704,W503,W504,B001,B006,B007,B008,B015,B017,B023,D100,D101,D102,D103,D104,D105,D107,D200,D201,D202,D205,D208,D209,D210,D211,D300,D400,D401,D402,D403,D412,D414,D419,E301,E302,E306,E401,E402,E711,E712,E722,E731,F401,F403,F405,F522,F523,F524,F541,F811,F821,F841,RST201,RST203,RST206,RST210,RST212,RST213,RST215,RST219,RST301,RST303,RST304,RST307,RST499,W291,W293
+FLAKE8_IGNORE = E121,E123,E126,E226,E24,E704,W503,W504
 
+FLAKE8_IGNORE += B001
+FLAKE8_IGNORE += B006
+FLAKE8_IGNORE += B007
+FLAKE8_IGNORE += B008
+FLAKE8_IGNORE += B015
+FLAKE8_IGNORE += B017
+FLAKE8_IGNORE += B023
+FLAKE8_IGNORE += D100
+FLAKE8_IGNORE += D101
+FLAKE8_IGNORE += D102
+FLAKE8_IGNORE += D103
+FLAKE8_IGNORE += D104
+FLAKE8_IGNORE += D105
+FLAKE8_IGNORE += D107
+FLAKE8_IGNORE += D200
+FLAKE8_IGNORE += D201
+FLAKE8_IGNORE += D202
+FLAKE8_IGNORE += D205
+FLAKE8_IGNORE += D208
+FLAKE8_IGNORE += D209
+FLAKE8_IGNORE += D210
+FLAKE8_IGNORE += D211
+FLAKE8_IGNORE += D300
+FLAKE8_IGNORE += D400
+FLAKE8_IGNORE += D401
+FLAKE8_IGNORE += D402
+FLAKE8_IGNORE += D403
+FLAKE8_IGNORE += D412
+FLAKE8_IGNORE += D414
+FLAKE8_IGNORE += D419
+FLAKE8_IGNORE += E301
+FLAKE8_IGNORE += E302
+FLAKE8_IGNORE += E306
+FLAKE8_IGNORE += E401
+FLAKE8_IGNORE += E402
+FLAKE8_IGNORE += E711
+FLAKE8_IGNORE += E712
+FLAKE8_IGNORE += E722
+FLAKE8_IGNORE += E731
+FLAKE8_IGNORE += F401
+FLAKE8_IGNORE += F403
+FLAKE8_IGNORE += F405
+FLAKE8_IGNORE += F522
+FLAKE8_IGNORE += F523
+FLAKE8_IGNORE += F524
+FLAKE8_IGNORE += F541
+FLAKE8_IGNORE += F811
+FLAKE8_IGNORE += F821
+FLAKE8_IGNORE += F841
+FLAKE8_IGNORE += RST201
+FLAKE8_IGNORE += RST203
+FLAKE8_IGNORE += RST206
+FLAKE8_IGNORE += RST210
+FLAKE8_IGNORE += RST212
+FLAKE8_IGNORE += RST213
+FLAKE8_IGNORE += RST215
+FLAKE8_IGNORE += RST219
+FLAKE8_IGNORE += RST301
+FLAKE8_IGNORE += RST303
+FLAKE8_IGNORE += RST304
+FLAKE8_IGNORE += RST307
+FLAKE8_IGNORE += RST499
+FLAKE8_IGNORE += W291
+FLAKE8_IGNORE += W293
+
+null  :=
+space := $(null) #
+comma := ,
+
+FLAKE8_IGNORE := $(subst $(space),$(comma),$(strip $(FLAKE8_IGNORE)))
```

We can use backslashes to split the variable in multiple lines. I also added quotes to account for the extra white spaces that it introduces, otherwise `flake8` will interpret the warnings as paths to look for linting.
```suggestion
FLAKE8_IGNORE = "E121,E123,E126,E226,E24,E704,W503,W504,\
	B001,\
	B006,\
	B007,\
	B008,\
	B015,\
	B017,\
	B023,\
	D100,\
	D101,\
	D102,\
	D103,\
	D104,\
	D105,\
	D107,\
	D200,\
	D201,\
	D202,\
	D205,\
	D208,\
	D209,\
	D210,\
	D211,\
	D300,\
	D400,\
	D401,\
	D402,\
	D403,\
	D412,\
	D414,\
	D419,\
	E301,\
	E302,\
	E306,\
	E401,\
	E402,\
	E711,\
	E712,\
	E722,\
	E731,\
	F401,\
	F403,\
	F405,\
	F522,\
	F523,\
	F524,\
	F541,\
	F811,\
	F821,\
	F841,\
	RST201,\
	RST203,\
	RST206,\
	RST210,\
	RST212,\
	RST213,\
	RST215,\
	RST219,\
	RST301,\
	RST303,\
	RST304,\
	RST307,\
	RST499,\
	W291,\
	W293,\
"
```

---

Yeah that works too, just a personal preference over which was is easier to read as I do not like how make files need the line break character, 🤷